rockfield/style-rtl.css

```diff
@@ -2850,7 +2850,7 @@ body:not(.fse-enabled) .site-description {
 	z-index: 1;
 }
 
-.main-navigation > div > ul li:hover, .main-navigation > div > ul li[focus-within] {
+.main-navigation > div > ul li:hover, .main-navigation.js-focus-within > div > ul li[focus-within], .js-focus-within .main-navigation > div > ul li[focus-within] {
```

Comment:
This PR changed some selectors accidentally I believe (see https://github.com/Automattic/themes/pull/8570)